python-package/lightgbm/dask.py

```diff
@@ -534,7 +594,16 @@ def to_local(self):
 class DaskLGBMRanker(LGBMRanker, _DaskLGBMModel):
     """Distributed version of lightgbm.LGBMRanker."""
 
-    def fit(self, X, y, sample_weight=None, init_score=None, group=None, client=None, **kwargs):
+    def fit(
+        self,
+        X: _DaskCollection,
```

Comment:
Same about inability to pass `dask Series` as above.

Reply:
fixed in https://github.com/microsoft/LightGBM/pull/3866/commits/8b19e04b05f83a38f5213c1103040f49d0a20988